solutions/usaco-811.mdx

```diff
@@ -25,18 +25,21 @@ Due to the low bounds on $N$ and $B$, we can run a [DFS (Depth First Search)](/s
 
 using namespace std;
 
+const int MAX_N = 250;
 int n;
 int m;
-int depths[251];
-bool vis[251][251];
-pair<int, int> shoes[251];
+vector<int> depths(MAX_N + 1);
+bool vis[MAX_N + 1][MAX_N + 1];
```

Comment:
uuuuh if you made the others vectors why not this one?

Reply:
vectors are weird for 2d arrays

Reply:
also i normally do a bool array for `visited`. if you want me to change it, ill do it :D

Reply:
really?
it's just `vector<vector<bool>> x(size, vector<bool>(size));`

Reply:
alr, ill do it after aoc